galpy/potential/FDMDynamicalFrictionForce.py

```diff
@@ -79,13 +63,10 @@ def FDMfactor(self, r, vs):
         if self._lnLambda:
             return self._lnLambda
         else:
-            kr = self.krValue(r, vs)
+            kr = 2 * self.krValue(r, vs)
+            I = -sp.sici(kr)[1] + np.log(kr) + 0.5772156649015329
```

Comment:
```suggestion
I = -sp.sici(kr)[1] + np.log(kr) + numpy.euler_gamma
```
Not a full review yet, but just noticed this...

Reply:
yes, thank you !